portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/GlobalSpecificComponents/Policies.tsx

```diff
@@ -0,0 +1,610 @@
+                    {/**
+                    * Breadcrumb Navigation.
+                    */}
+                    <Grid item md={12}>
+                        <div className={classes.titleWrapper}>
+                            <Link to='/global-policies' className={classes.titleLink}>
+                                <Typography variant='h4' component='h1'>
+                                    <FormattedMessage
+                                        id='globalPolicies.heading'
+                                        defaultMessage='Global Policies'
+                                    />
+                                </Typography>
+                            </Link>
+                            <Icon>keyboard_arrow_right</Icon>
+                            <Typography variant='h4' component='h2'>
+                                {isCreateNew ? 
+                                    <FormattedMessage
+                                        id='globalPolicies.create.create.heading'
+                                        defaultMessage='Create A New Global Policy'
+                                    />
+                                    : 
+                                    <FormattedMessage
+                                        id='globalPolicies.create.edit.heading'
+                                        defaultMessage='Edit Global Policy'
+                                    />
+                                }
+                            </Typography>
+                        </div>
+                    </Grid>
+
+                    {/**
+                    * Name & Description Fields.
+                    */}
+                    <Box mb={2}>
+                        <TextField
+                            fullWidth
+                            required
+                            id='outlined-required'
+                            label='Name'
+                            variant='outlined'
+                            value={name}
+                            onChange={handleNameChange}
+                            className={classes.textField}
+                        />
+                    </Box>
+                    <Box mb={2}>
+                        <TextField
+                            fullWidth
+                            required
+                            id='outlined-multiline-static'
+                            label='Description'
+                            multiline
+                            rows={3}
+                            variant='outlined'
+                            value={description}
+                            onChange={handleDescriptionChange}
+                            className={classes.textField}
+                        />
+                    </Box>  
+                    
+                    <Box className={classes.operationListingBox}>  
+                        {/**
+                        * Left side panel where we can droped policies.
```

Comment:
```suggestion
                        * Left side panel where we can drop policies.
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)